### PR TITLE
Favicon updated

### PR DESCRIPTION
**Pull request by Yash621**
I have updated the favicon, please review my PR.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/361"></a>

### CODE REVIEW

**Comment by CodingSpecies** [comment]
@Yash621 did you add a new favicon? If so please can you add a screenshot of it please? Thanks.

**Comment by CodingSpecies** [comment]
Or did you rename the file?

File: public/manifest.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "Create React App Sample",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "newFavicon.png",
```
Comment (rohan-kulkarni-25) [inline public/manifest.json:6]: - You need to have `.ico` extension if you are adding the image as fav icon

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/manifest.json (hunk 1 above)
Comment (rohan-kulkarni-25) [reply public/manifest.json:6]: - Please Add screen shot of the image which you will like to add

Comment (Yash621) [reply public/manifest.json:6]: ohh okay !! sure

**Pull request closed without merging**